.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install requirements
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: First test
+      run: |
+        python setup.py install
+        python -m nafas test
+    - name: Install dev-requirements
+      run: |
+        pip install --upgrade --upgrade-strategy=only-if-needed -r dev-requirements.txt
+    - name: Version check
+      run: |
+        python otherfiles/version_check.py
+    - name: Test with pytest
+      run: |
+        python -m pytest test --cov=nafas --cov-report=term
+    - name: Other tests
+      run: |
+        python -m vulture nafas/ setup.py --min-confidence 65 --exclude=__init__.py --sort-by-size
+        python -m bandit -r nafas -s B311
+        python -m pydocstyle --match-dir=nafas -v
+    - name: Codecov
+      run: |
+        codecov
```

Comment:
new line may be forgotten here.

Reply:
solved in https://github.com/sepandhaghighi/nafas/pull/25/commits/5e4c0f44d37b4cdbafdc9200ec1f66ccc591db44